blog/2021-07-17-introducing-metriql.md

```diff
@@ -0,0 +1,94 @@
+
+However; Looker is still an enterprise product just like Tableau and PowerBI and tries to solve multiple problems at once. Luckily, with Looker’s wave, new data modeling tools have emerged such as dbt Core. It’s an open-source data transformation tool at its core but you can also test and document your data, just like LookML. It’s not comparable to LookML though, it has a much broader audience because it enables all the other data tools to use the generated data models in your data warehouse. It's [not designed](https://blog.getdbt.com/how-do-you-decide-what-to-model-in-dbt-vs-lookml/) to be the serving layer unlike LookML. With dbt Core, we believe that the data transformation layer is a solved problem but the serving layer for your data is still missing. The data tools connect directly to the data warehouses 
+
+> metriql is LookML for all the BI tools in the market. You transform, test, and document your data with dbt, define your metrics with metriql and serve data models to your data tools in a consistent way.
+
+metriql is an open-source project that lets you define your company metrics as code in a single central metric repository using dbt and later let you sync the data models to all your data tools at once.
+
+metriql sits between your data-warehouse and the data tools. It doesn’t have its query engine, instead leverages your existing data warehouse. While it has a REST API, it’s often not enough. A more tight integration was required for the BI tools so we built MQL, a SQL dialect for your metriql datasets. We have native integrations with multiple tools such as Google Sheets and Google Data Studio and use Trino-interface for some others such as Looker & Tableau.
+
+If you don't have a standardized way of sharing the data across your organization and tools, each team ends up introducing their own way of understanding the data and it gets messy and painful to maintain and this bears discussions like: [self serve is a problem](https://benn.substack.com/p/self-serve-still-a-problem). Our goal is to contribute to the solution of this self-serve problem by working on standardization of the metric definition and distribution.
+
+What we came to understanding of by working with data-driven and succesful companies in the past years is that they use the right tool for the right, they don't try to force one tool to fit all use cases. BI tools are now the front-end of your data and your business stakeholders should have the option to choose any front-end depending on their preferences and needs. While the Headless BI concept is relatively new and still in the early stages, it has many use-cases that saves the data teams time just like how dbt started to replace data transformation within the BI tools. Happy and excited to be one of the first companies in this space! 🤞
```

Comment:
it has many use-cases that save the data teams time just like how dbt started to replace data transformation within the BI tools. 
Burda how dbt saved data teams time by replacing data transformation withing BI tools mu demek istiyorsun yoksa baska bir sey mi kast ediyorsun?

Reply:
aynen dedigini kastediyorum